icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/PluralRulesTest.java

```diff
@@ -399,7 +399,10 @@ public void testUniqueRules() {
                             }
                             String old = collisionTest.get(item);
                             if (old != null) {
-                                errln(locale + "\tNon-unique rules: " + item + " => " + old + " & " + foundKeyword);
+                                if (!locale.getLanguage().equals("fr") ||
+                                        !logKnownIssue("21328", "fr Non-unique rules: 1e6 => one & many0")) {
```

Comment:
This looks fine to me.  I marked the new ticket ICU-21328 and the ticket ICU-21322 (replace FixedDecimal with DecimalQuantity for plural rule samples) as related.